templates/policies/sts_installer_permission_policy.json

```diff
@@ -177,7 +177,17 @@
                 "ec2:DescribeVpcEndpointServiceConfigurations",
                 "ec2:DescribeVpcEndpointServicePermissions",
                 "ec2:DescribeVpcEndpointServices",
-                "ec2:ModifyVpcEndpointServicePermissions"
+                "ec2:ModifyVpcEndpointServicePermissions",
+                "iam:CreateRole",
+                "iam:DeleteRole",
+                "iam:DeleteRolePolicy",
+                "iam:PutRolePolicy",
+                "s3:GetBucketReplication",
+                "iam:DeleteAccessKey",
+                "iam:DeleteUser",
+                "iam:DeleteUserPolicy",
+                "iam:GetUserPolicy",
+                "iam:PutUserPolicy"
```

Comment:
I think these should all be sorted by API group (`iam` vs `s3` vs `ec2`) and alphabetically under that

Reply:
These too are likely not needed and are there to support the CloudCredential operator which is in "manual" mode for STS.